ThunderTable/TableViewController.swift

```diff
@@ -136,26 +136,71 @@ open class TableViewController: UITableViewController {
         tableView.register(defaultNib, forCellReuseIdentifier: "Cell")
     }
 	
-	private var dynamicChangeObserver: NSObjectProtocol?
-	
-	public var shouldRedrawWithContentSizeChange = true
+    private var dynamicChangeObserver: NSObjectProtocol?
+    
+    private var accessibilityObservers: [Any] = []
+    
+    public var adjustsFontForContentSizeCategory: Bool = true
```

Comment:
Docs for these please :D

Reply:
Docs should be available from the protocol that this is a variable from I believe :D

Reply:
Checked, and apparently that is not true!